regression/cbmc-library/__builtin_clz-02/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+For this example, __builtin_clz is fully evaluated in the front-end. This test
+ensures that the function call no longer shows up.
```

Comment:
Does it actually check that the function call is gone?

Reply:
Thank you for pointing out this outdated comment! This is now fixed.